pkg/master/ports/ports.go

```diff
@@ -49,4 +50,7 @@ const (
 	// CloudControllerManagerPort is the default port for the cloud controller manager server.
 	// This value may be overridden by a flag at startup.
 	CloudControllerManagerPort = 10258
+	// SchedulerPort is the default secure port for the scheduler status server.
+	// May be overridden by a flag at startup.
+	SchedulerPort = 10259
```

Comment:
can we name this SecureSchedulerPort to avoid references silently switching which one they refer to?

Reply:
@liggitt sounds reasonable, will update soon.